server/re/src/icatGeneralMS.cpp

```diff
@@ -121,6 +121,8 @@ msiQuota( ruleExecInfo_t *rei ) {
  *
  * \since pre-2.1
  *
+ * \deprecated Deprecated in 4.3.2. Use alternative, such as msiSetDefaultResource(...) instead.
```

Comment:
This assumes `msiSetDefaultResource` works. Does it?

Reply:
True, mostly threw it in there as a 'it would be nice to suggest a different solution'. Can drop it, since I'm not sure that it would cover the same case.

Reply:
I agree that offering an alternative is good. We also have to prove that use-case is indeed covered though.

Please investigate and share what you find here.

Reply:
And check the name. I think it’s either Def or Resc.

Reply:
`msiSetDefaultResc`!

Reply:
I think it's better to drop the suggestion. At the very least, they do set different properties, i.e., `rei->doi->rescName` vs `rei->rescName` if I recall correctly.

Reply:
Agreed. Maybe just say "Its use should be avoided." or something to that effect

Citing the fancy new docs: https://github.com/irods/irods_docs/blob/main/docs/developers/deprecation.md#how-do-i-deprecate-a-feature

Reply:
Done.